internal/provider/resource_gitlab_application_settings_test.go

```diff
@@ -0,0 +1,33 @@
+//go:build acceptance
+// +build acceptance
+
+package provider
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestAccGitlabApplicationSettings_basic(t *testing.T) {
```

Comment:
What about adding an EE only test for the fields only available in premium and above.  One thing we also have to think about in the future is if we end up building out the acceptance tests some more unless we do it in a non-parallel fashion and revert all of the tested settings to their initial state after testing then those settings could potentially influence other acceptance tests

Reply:
Yes, you are right - I think for other resources it makes a lot of sense and is easy to do. Given that this resource is kinda experimental and I don't really know where this is going or how much maintenance burden it is I'd like to keep it as-is.

However, I completely agree that we should think about what do to if we change settings here or other resources which affect the entire instance ...